### DISCUSSION TITLE
csdm database does not exist

### DISCUSSION THREAD

**Discussion by RichardHenri** [akiver/cs-demo-manager#719] [Q&A]
Hello,

I'm trying to install CSDM 3.0.7 but when I try to connect, it says that the csdm database does not exist. Postgresql 16 is correctly installed if I'm not mistaken (service is running) and I don't know what is wrong with this. Any ideas?

![image](https://github.com/akiver/cs-demo-manager/assets/45435926/3005fb14-59ca-4778-9e92-78218b1077ed)

**Comment by akiver** [comment]
The database should be created if necessary when the app starts, I don't know why it's not the case for you.  

Could you please upload the log files (it's located in `C:/Users/USERNAME/.csdm/logs`)?

You could also try to create the database manually:

1. `psql -h 127.0.0.1 -U postgres -p 5432`
2. `create database csdm`

**Reply by RichardHenri** [reply]
[csdm.log](https://github.com/akiver/cs-demo-manager/files/13908317/csdm.log)

Here's the log :)

Creating the database manually does not seem to work either.

**Reply by akiver** [reply]
Do you have a working connection using `psql -h 127.0.0.1 -U postgres -p 5432`?
If so what's the output of the command `\list`?
I noticed that the last log's date is the 31 of December, are you sure you tried to start the app after creating the database manually?

**Reply by RichardHenri** [reply]
Seems like it. This is what I get: 

![image](https://github.com/akiver/cs-demo-manager/assets/45435926/43be842e-c061-4346-a9d2-2a51244e7464)

**Reply by akiver** [reply]
Do you fully restart the app from the system tray? If you don't the app will not try to create the database as it's done only on app startup.

The database doesn't exist - you can try to create it manually with `create database csdm` and verify that it has been created with `\list`.
